### PR TITLE
Add opt ports to helm

### PR DESCRIPTION
**Pull request by psheets**
Added `extraPorts` option to brokers, services, controllers, minions and associated services. 

Used the same pattern as `extraEnvs`

### CODE REVIEW

File: kubernetes/helm/pinot/templates/controller/service.yaml [hunk 1]
```diff
@@ -31,5 +31,11 @@ spec:
     # [pod_name].[service_name].[namespace].svc.cluster.local
     - name: {{ .Values.controller.service.name }}
       port: {{ .Values.controller.service.port }}
+    {{- if .Values.controller.service.extraPorts }}
```
Comment (abhioncbr) [inline kubernetes/helm/pinot/templates/controller/service.yaml:34]: Minor, just for consistency, add `if` to all the other services for extra ports. Thanks

**Comment by psheets** [comment]
Are there any other changes required before merging?

**Comment by Jackie-Jiang** [comment]
@xiangfu0 Can you please help review the change?

**Review by xiangfu0 — APPROVED** [review]
lgtm, thanks for your contribution!

**Pull request merged**